Archipelago.cpp

```diff
@@ -241,34 +241,46 @@ void AP_SetClientVersion(AP_NetworkVersion* version) {
 }
 
 void AP_SendItem(int64_t idx) {
-    printf("AP: Checked '%s'. Informing Archipelago...\n", getLocationName(ap_game,idx).c_str());
+    AP_SendItem(std::vector<int64_t>{ idx });
+}
+void AP_SendItem(std::vector<int64_t> const& locations) {
+    printf("AP: ");
+    for (int64_t idx : locations) {
+        printf("Checked '%s'.", getLocationName(ap_game, idx).c_str());
+    }
+    printf("Informing Archipelago...\n", locations.size());
```

Comment:
`locations.size` unused? This print line can be removed entirely though, doesnt really add anything